core/server/api/v2/utils/index.js

```diff
@@ -9,5 +9,16 @@ module.exports = {
 
     get validators() {
         return require('./validators');
+    },
+
+    /**
+     *  TODO: We need to check for public context as permission stage overrides
+     * the whole context object currently: https://github.com/TryGhost/Ghost/issues/10099
+     */
+    isContentAPI: (frame) => {
```

Comment:
We know it's not optimal putting this helper fn here. It should not live in index.js, but let's accept for now 👍